otter/convergence/model.py

```diff
@@ -14,31 +14,95 @@
 
 
 class CLBNodeCondition(Names):
-    """Constants representing the condition a load balancer node can be in"""
-    ENABLED = NamedConstant()   # Node can accept new connections.
-    DRAINING = NamedConstant()  # Node cannot accept any new connections.
-                                # Existing connections are forcibly terminated.
-    DISABLED = NamedConstant()  # Node cannot accept any new connections.
-                                # Existing connections are permitted to continue.
+    """
```

Comment:
I'm not going to make a blocking review comment, but for the record I think it would make more sense to put :ivar: directives into the class docstring. ( http://sphinx-doc.org/domains.html#info-field-lists ).